refinery/ui/source/js/dashboard/directives/data-set-preview.html

```diff
@@ -15,14 +25,41 @@
         </button>
       </div>
       <div class="floatbox-max-left">
+        <button
+          id="back-to-satori"
+          class="btn btn-super-important clickable"
+          ng-click="preview.close()"
+          ng-if="preview.exploration">
+          <i class="fa fa-angle-double-left"></i>
+          Back to exploration
+        </button>
+        <span class="separator" ng-if="preview.exploration">&middot;</span>
         <a
+          id="view-in-file-browser"
           class="btn btn-important"
+          ng-show="!preview.settings.djangoApp.satoriDemo"
           ng-href="data_sets/{{ preview.dataSetUuid }}">
           <i class="fa fa-table"></i>
           View content in File Browser
         </a>
+        <a
+          id="view-in-file-browser"
+          class="btn btn-important is-not-allowed"
```

Comment:
is-not-allowed seems like an odd name for a class, once again preference, it's fine. It's only in the dashboard.css file.